DashAI/front/src/components/experiments/ExperimentsTable.jsx

```diff
@@ -191,6 +197,8 @@ function ExperimentsTable({
             },
           },
         }}
+        sortModel={[{ field: "id", sort: "desc" }]}
+        columnVisibilityModel={{ id: false }}
```

Comment:
Is this to have the ids of each row when handling an action?

Reply:
I added the id to sort the rows so that when a new experiment is added it will always be positioned at the beginning of the table.